src/components/navigation/AppNavigationBoard.vue

```diff
@@ -36,11 +36,15 @@
 					<button v-if="board.acl.length === 0"
 						class="icon-shared"
 						style="opacity: 0.3"
+						:title="'Share'"
```

Comment:
Please make those translatable.

Reply:
@juliushaertl  sir, I have done the changes, please have a look.